wrench-sample/src/main/java/com/example/wrench/MainActivity.java

```diff
@@ -17,44 +18,63 @@
 
     private ActivityMainBinding activityMainBinding;
 
+    private WrenchSampleViewModel wrenchSampleViewModel;
```

Comment:
I would move this and `private ActivityMainBinding activityMainBinding;` to local methods.
Since it's not used outside `onCreate`